anitab-org/bridge-in-tech-web
# Feat: Update user personal background

### Description
Allow user to update their personal background

Fixes #49 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- login then go to `My Space` > 'Personal Details' > `Personal Background`
- If this is the first time user view the Personal Background page, it will show the message below:
<img width="493" alt="Screen Shot 2020-08-02 at 10 08 40 pm" src="https://user-images.githubusercontent.com/29667122/89123502-32b46700-d513-11ea-8ce2-85c14f55ebdd.png">

- Fill in the personal background information. On successful process, see message below
<img width="451" alt="Screen Shot 2020-08-04 at 8 34 52 am" src="https://user-images.githubusercontent.com/29667122/89234001-41dc0780-d62e-11ea-8b4b-b4bce3c84a00.png">

If user click `Save` button without selecting anything, the default value is saved "Prefeer not to say"

<img width="473" alt="Screen Shot 2020-08-04 at 8 35 19 am" src="https://user-images.githubusercontent.com/29667122/89234086-7a7be100-d62e-11ea-8387-c108a530197a.png">


- Now change the details and on successful update see message below

<img width="464" alt="Screen Shot 2020-08-04 at 8 35 45 am" src="https://user-images.githubusercontent.com/29667122/89234093-810a5880-d62e-11ea-9d86-4376669705fc.png">

### Checklist:

**Code/Quality Assurance Only**
